src/app/core/bible_metadata.py

```
@@ -0,0 +1,72 @@
+from typing import List, Dict
```

Ruff (0.15.17)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)